### DISCUSSION TITLE
Some rules you can use:

### DISCUSSION THREAD

**Discussion by elucian** [ykdojo/defaang#213] [General]
1. Use only English to chat;
2. Treat everyone with respect;
3. Chat in the right channel;
4. Don't post large blocks of code;
5. Don't spam people asking for help;
6. No politics or religious talk;
7. No piracy, swearing  or hate;
8. Set status "online" to chat.

**Comment by epicadidash** [comment]
**Copied from somewhere I know.**

**Reply by epicadidash** [reply]
@elucian I also did so but @ykdojo  told me not to copy

**Reply by elucian** [reply]
Yes. I do copy paste all the time.

**Comment by ykdojo** [comment]
Talked about this on stream.

I'd prefer to grow the rules slowly and carefully.

For now, this PR should cover pretty much all of those points: https://github.com/ykdojo/defaang/pull/217

**Reply by elucian** [reply]
Sure. What was I thinking. Please ignore.
